### PR TITLE
update ssloption suppressHandshakeErrors documentation

### PR DESCRIPTION
**Pull request by billw1024**

### CODE REVIEW

File: dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -25,7 +25,7 @@ ssl.sessionTimeout=Session timeout
 ssl.sessionTimeout.desc=Amount of time to wait for a read or write request to complete on a socket. This value is overridden by protocol-specific timeouts.
 
 suppressHandshakeErrors=Suppress SSL handshake errors
-suppressHandshakeErrors.desc=Disable logging of SSL handshake errors. SSL handshake errors can occur during normal operation, however these messages can be useful when SSL is behaving unexpectedly.
+suppressHandshakeErrors.desc=Disable logging of SSL handshake errors. SSL handshake errors can occur during normal operation, however these messages can be useful when SSL is behaving unexpectedly. If disabled the message and console logs will not record handshake errors, and only the trace log will record handshake errors when SSL Channel tracing is on.
```
Comment (helyarp) [inline dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties:28]: Change:
> If disabled the message and console logs will not record handshake errors, and only the trace log will record handshake errors when SSL Channel tracing is on.

to:
> If disabled, the message and console logs do not record handshake errors, and the trace log records handshake errors only when SSL Channel tracing is on.

Comment (billw1024) [reply dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties]: made suggested update

File: dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -25,7 +25,7 @@ ssl.sessionTimeout=Session timeout
 ssl.sessionTimeout.desc=Amount of time to wait for a read or write request to complete on a socket. This value is overridden by protocol-specific timeouts.
 
 suppressHandshakeErrors=Suppress SSL handshake errors
-suppressHandshakeErrors.desc=Disable logging of SSL handshake errors. SSL handshake errors can occur during normal operation, however these messages can be useful when SSL is behaving unexpectedly.
+suppressHandshakeErrors.desc=Disable logging of SSL handshake errors. SSL handshake errors can occur during normal operation, however these messages can be useful when SSL is behaving unexpectedly. If disabled, the message and console logs do not record handshake errors, and the trace log will record handshake errors when SSL Channel tracing is on.
```
Comment (helyarp) [inline dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties:28]: Change "will record" to "records"

Comment (billw1024) [reply dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties]: record change made.

**Comment by billw1024** [comment]
#build

**Comment by billw1024** [comment]
#run-libby-bot

**Pull request merged**